src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -523,7 +525,10 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
                             {...inviteCollabsEmailTracking}
                         >
                             {this.renderInviteePermissionsDropdown()}
-                            {showUpgradeOptions && !showUpgradeInlineNotice && this.renderUpgradeLinkDescription()}
+                            {showUpgradeOptions &&
+                                !showUpgradeInlineNotice &&
+                                !showUpsellInlineNotice &&
```

Comment:
this will have the same effect as the line above since we are migrating from an older inline notice (which) showUpgradeInlineNotice is using) to a new one (which showUpsellInlineNotice is using).